go/vt/servenv/grpc_auth.go

```diff
@@ -33,7 +33,7 @@ var grpcAuthServerFlagHooks []func(*pflag.FlagSet)
 // RegisterGRPCServerAuthFlags registers flags required to enable server-side
 // authentication in vitess gRPC services.
 //
-// `go/cmd/*`` entrypoints should call this function before
+// `go/cmd/*“ entrypoints should call this function before
```

Comment:
nit: looks like unrelated change.

Reply:
Was due to pre-commit hook forcing me to accept these formatting changes. Should I run with `git commit --no-verify`, or maybe my local tooling is out of date?

Reply:
so, to be clear,

```suggestion
// `go/cmd/*` entrypoints should call this function before
```